src/Utilities/DoubleDicts.jl

```diff
@@ -72,6 +73,18 @@ end
     v
 end
 
+# reversing IndexDoubleDict is ok because they map CI to CI
+function MOIU._reverse_dict(
+    dest::IndexDoubleDict{DI}, src::IndexDoubleDict{DI}
+) where DI
+    for (k, v) in src.dict
+        dest.dict[k] = DI(values(v) .=> keys(v))
```

Comment:
Why not call `MOIU._reverse_dict(dest.dict[k], v)` ? That way, it would automatically specialize for the best way to reverse the inner dicts as well.

Reply:
I will add a single input version, because `dest.dict[k]` is initially empty and its more efficient to pass data on construction instead of building an empty one and then populating.